### DISCUSSION TITLE
Possible to disable auto/remote updates?

### DISCUSSION THREAD

**Discussion by ProgramDeez** [runew0lf/RuinedFooocus#149] [Q&A]
Hey there! I have a use case that works best using a static version of RF. 

Is it possible to turn off auto updates or anyway to easily workaround? 

Would still like to be able to update in the future, just would be better to have the flexibility to control when updates occur. Thank you in advance!

**Comment by runew0lf** [comment]
Nope sorry if you've made any changes then its better to use a fork and do it there!
